[Automation] native-image run fails for io.quarkus:quarkus-junit-common:3.37.0.CR1

## native-image run on ubuntu-latest / JDK 25 / mode current-defaults

Failure kind: native-image run
OS: ubuntu-latest
JDK: 25
Native-image mode: current-defaults
Reproducer: `env GVM_TCK_LV=3.37.0.CR1 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean test -Pcoordinates=io.quarkus:quarkus-junit-common:3.33.1`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/27299130550
Last 300 lines of the log:
```console
[native-image-plugin] GraalVM location read from environment variable: GRAALVM_HOME
[native-image-plugin] Native Image executable path: /opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/bin/native-image
Picked up NATIVE_IMAGE_OPTIONS: -H:BuildOutputJSONFile=/tmp/native-image-build-output.json
Warning: Invalid classpath entry: /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/resources/test
Warning: The option '-H:BuildOutputJSONFile=/tmp/native-image-build-output.json' is experimental and must be enabled via '-H:+UnlockExperimentalVMOptions' in the future.
Warning: Please re-evaluate whether any experimental option is required, and either remove or unlock it. The build output lists all active experimental options, including where they come from and possible alternatives. If you think an experimental option should be considered as stable, please file an issue.
Apply jar:file:///opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/library-support.jar!/META-INF/native-image/com.oracle.svm/thirdparty/native-image.properties
Apply jar:file:///opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/library-support.jar!/META-INF/native-image/com.oracle.svm/polyglot/native-image.properties
Executing [
DRIVER_TEMP_DIR_ENV_VARIABLE=/tmp/driverRoot-4723970748988139367 \
HOME=/home/runner \
LANG=C.UTF-8 \
PATH=/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/bin:/opt/hostedtoolcache/Java_GraalVM_jdk/21/x64/bin:/snap/bin:/home/runner/.local/bin:/opt/pipx_bin:/home/runner/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.ghcup/bin:/home/runner/.dotnet/tools:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin \
PWD=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1 \
USE_NATIVE_IMAGE_JAVA_PLATFORM_MODULE_SYSTEM=true \
/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/bin/java \
-XX:+UnlockExperimentalVMOptions \
-XX:+EnableJVMCI \
-Dtruffle.TrustAllTruffleRuntimeProviders=true \
-Dtruffle.TruffleRuntime=com.oracle.truffle.api.impl.DefaultTruffleRuntime \
-Dgraalvm.ForcePolyglotInvalid=true \
-Dgraalvm.locatorDisabled=true \
-Dsubstratevm.HostLibC=glibc \
--add-exports=java.base/com.sun.crypto.provider=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.access=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.event=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.foreign=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.loader=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.logger=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.misc=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.objectfile,org.graalvm.nativeimage.pointsto \
--add-exports=java.base/jdk.internal.module=org.graalvm.nativeimage.base,org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.perf=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.platform=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.ref=org.graalvm.nativeimage.builder,org.graalvm.nativeimage.objectfile \
--add-exports=java.base/jdk.internal.reflect=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.util=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.vm.annotation=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal.vm=org.graalvm.nativeimage.builder \
--add-exports=java.base/jdk.internal=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.invoke.util=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.net.www=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.net=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.nio.ch=org.graalvm.nativeimage.builder,org.graalvm.nativeimage.objectfile \
--add-exports=java.base/sun.reflect.annotation=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.reflect.generics.factory=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.reflect.generics.reflectiveObjects=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.reflect.generics.repository=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.reflect.generics.scope=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.reflect.generics.tree=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.security.jca=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.security.provider=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.security.rsa=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.security.ssl=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.security.util=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.security.x509=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.text.spi=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.util.calendar=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.util.cldr=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.util.locale.provider=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.util.locale=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.util.resources=org.graalvm.nativeimage.builder \
--add-exports=java.base/sun.util=org.graalvm.nativeimage.builder \
--add-exports=java.management/com.sun.jmx.mbeanserver=org.graalvm.nativeimage.builder \
--add-exports=java.management/sun.management=org.graalvm.nativeimage.builder,org.graalvm.nativeimage.pointsto \
--add-exports=jdk.graal.compiler/jdk.graal.compiler.util.json=com.oracle.graal.reporter \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.aarch64=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.objectfile \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.amd64=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.objectfile \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.code.site=com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.code.stack=jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.code=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.objectfile,org.graalvm.nativeimage.pointsto,org.graalvm.truffle.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.common=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.pointsto \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot.aarch64=jdk.graal.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot.amd64=jdk.graal.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot.riscv64=jdk.graal.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.hotspot=com.oracle.graal.graal_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.meta=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.base,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.configure,org.graalvm.nativeimage.objectfile,org.graalvm.nativeimage.pointsto,org.graalvm.truffle.compiler \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.riscv64=com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.runtime=com.oracle.graal.graal_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.pointsto \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.services=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.events=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.event=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.jfc=com.oracle.svm.svm_enterprise,org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.settings=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.tracing=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal=com.oracle.svm.svm_enterprise,org.graalvm.nativeimage.builder \
--add-exports=jdk.management/com.sun.management.internal=org.graalvm.nativeimage.builder \
-XX:+UseJVMCINativeLibrary \
-Xss10m \
-XX:+UseParallelGC \
-XX:MaxRAMPercentage=85.0 \
'-Dsvm.build.memoryUsageReasonText=$CI set to '"'"'true'"'"'' \
-XX:GCTimeRatio=9 \
-XX:+ExitOnOutOfMemoryError \
-Djava.awt.headless=true \
'-Dorg.graalvm.vendor=Oracle Corporation' \
-Dorg.graalvm.vendorurl=https://www.graalvm.org/ \
'-Dorg.graalvm.vendorversion=Oracle GraalVM 25.0.3+9.1' \
-Dorg.graalvm.version=25.0.3 \
-Dcom.oracle.graalvm.isaot=true \
-Djava.system.class.loader=com.oracle.svm.hosted.NativeImageSystemClassLoader \
-Dorg.graalvm.nativeimage.imagecode=buildtime \
-Xshare:off \
-Djdk.internal.lambda.disableEagerInitialization=true \
-Djdk.internal.lambda.eagerlyInitialize=false \
-Djava.lang.invoke.InnerClassLambdaMetafactory.initializeLambdas=false \
-Djava.lang.invoke.MethodHandle.DONT_INLINE_THRESHOLD=-1 \
-Djava.lang.invoke.MethodHandle.PROFILE_GWT=false \
-Djunit.platform.listeners.uid.tracking.output.dir=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/test-results/test/testlist \
--add-modules=ALL-DEFAULT \
--enable-native-access=org.graalvm.nativeimage.objectfile,org.graalvm.nativeimage.pointsto,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.configure,org.graalvm.nativeimage.base,org.graalvm.nativeimage.foreign,com.oracle.svm.svm_enterprise,com.oracle.graal.reporter,org.graalvm.espresso.shared.svm,org.graalvm.nativeimage.shaded.capnproto \
--module-path \
/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/builder/svm-configure.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/builder/espresso-svm.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/builder/native-image-base.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/builder/reporter.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/builder/svm-foreign.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/builder/svm.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/builder/svm-capnproto-runtime.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/builder/svm-enterprise.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/builder/objectfile.jar:/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/builder/pointsto.jar \
-Dsvm.modulesupport.addedModules=java.rmi,java.sql,org.graalvm.collections,jdk.jdi,java.logging,java.xml.crypto,java.transaction.xa,java.xml,jdk.jfr,jdk.internal.vm.ci,java.datatransfer,org.graalvm.truffle.compiler,jdk.internal.md,java.desktop,java.naming,jdk.internal.ed,java.prefs,java.net.http,jdk.compiler,java.security.sasl,jdk.internal.opt,org.graalvm.nativeimage,jdk.zipfs,jdk.attach,java.base,jdk.management.agent,org.graalvm.word,jdk.internal.le,java.management,java.sql.rowset,jdk.unsupported,jdk.jlink,jdk.jdwp.agent,jdk.internal.jvmstat,java.instrument,java.security.jgss,jdk.management,java.compiler,java.scripting,jdk.graal.compiler,jdk.jdeps,java.management.rmi \
--add-modules=java.rmi,java.sql,org.graalvm.collections,jdk.jdi,java.logging,java.xml.crypto,java.transaction.xa,java.xml,jdk.jfr,jdk.internal.vm.ci,java.datatransfer,org.graalvm.truffle.compiler,jdk.internal.md,java.desktop,java.naming,jdk.internal.ed,java.prefs,java.net.http,jdk.compiler,java.security.sasl,jdk.internal.opt,org.graalvm.nativeimage,jdk.zipfs,jdk.attach,java.base,jdk.management.agent,org.graalvm.word,jdk.internal.le,java.management,java.sql.rowset,jdk.unsupported,jdk.jlink,jdk.jdwp.agent,jdk.internal.jvmstat,java.instrument,java.security.jgss,jdk.management,java.compiler,java.scripting,jdk.graal.compiler,jdk.jdeps,java.management.rmi \
--module \
org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner \
-keepalive \
/proc/6098/comm \
-imagecp \
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/libs/io.quarkus.quarkus-junit-common_tests.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.graalvm.buildtools/junit-platform-native/1.0.0/2974f834a2f0a0105f2284fba7ef16dc4d60de56/junit-platform-native-1.0.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/io.quarkus/quarkus-junit-common/3.37.0.CR1/dba62fabc1e87069b8a35824fc801f9403d75b9e/quarkus-junit-common-3.37.0.CR1.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter/6.1.0/52aef21f8c9fa491ff7eb20008e39102a8479f79/junit-jupiter-6.1.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-params/6.1.0/6e6dcbd963a6c16217350535a640e1c0fde0d619/junit-jupiter-params-6.1.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-engine/6.1.0/82414ee1a0308d2f096c37d317b93b66c13dbb9a/junit-jupiter-engine-6.1.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-api/6.1.0/5376d10370f32af4f3b1b7a378906ee58dcdacb3/junit-jupiter-api-6.1.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-reporting/6.1.0/914dbb41a35cf269e575037eca4b201bd5a803bb/junit-platform-reporting-6.1.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-launcher/6.1.0/78124de837a3e0b9e4e32a2c80ba47117f7f8c60/junit-platform-launcher-6.1.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.vintage/junit-vintage-engine/6.1.0/af18e7c935d0b0a31f484adab7d9d15083b15b09/junit-vintage-engine-6.1.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-engine/6.1.0/754271e8493a0be2838f3500b30971969b8603d0/junit-platform-engine-6.1.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-commons/6.1.0/3bb0447338b81f4396648c406d3e1c22f65f689/junit-platform-commons-6.1.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-console/6.1.0/b4785a1945051b8f5fa2daa82b72ea0f7188c47e/junit-platform-console-6.1.0.jar:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/tck-test-support/build/libs/tck-test-support-1.0.0-SNAPSHOT.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.assertj/assertj-core/3.22.0/c300c0c6a24559f35fa0bd3a5472dc1edcd0111e/assertj-core-3.22.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.jboss.logging/jboss-logging/3.6.3.Final/1cc9f976725720bb4a66f80af3e3aa6b9890d969/jboss-logging-3.6.3.Final.jar:/home/runner/.gradle/caches/modules-2/files-2.1/junit/junit/4.13.2/8ac9e16d933b6fb43bc7f576336b8f4d7eb5ba12/junit-4.13.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.opentest4j/opentest4j/1.3.0/152ea56b3a72f655d4fd677fc0ef2596c3dd5e6e/opentest4j-1.3.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.opentest4j.reporting/open-test-reporting-tooling-spi/0.2.5/92a64a32e3fbdfc094d05a714d88c619c0d12f38/open-test-reporting-tooling-spi-0.2.5.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-core/1.3/42a25dc3219429f0e5d060061f71acb49bf010a0/hamcrest-core-1.3.jar:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/classes/java/test:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/resources/test:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/test-results/test/testlist \
-imagemp \
/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/library-support.jar \
-H:CLibraryPath=/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/clibraries/linux-amd64/glibc,/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/clibraries/linux-amd64,/opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/clibraries \
-H:Path@driver=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/native/nativeTestCompile \
-H:BuildOutputJSONFile@user=/tmp/native-image-build-output.json \
-H:FallbackThreshold@user+api=0 \
-H:Name@user+api=io.quarkus.quarkus-junit-common_tests-tests \
-H:ConfigurationFileDirectories@user=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/native/generated/generateTestResourcesConfigFile,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.jupiter/junit-jupiter-engine/5.8.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.platform/junit-platform-engine/1.10.4,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.opentest4j/opentest4j/1.2.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.platform/junit-platform-commons/1.8.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.jupiter/junit-jupiter-params/5.9.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.platform/junit-platform-reporting/1.9.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.platform/junit-platform-launcher/1.9.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.vintage/junit-vintage-engine/5.9.1,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/junit/junit/4.13.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.hamcrest/hamcrest-core/1.3,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/io.quarkus/quarkus-junit-common/3.33.1,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.jboss.logging/jboss-logging/3.5.0.Final \
-H:Optimize@user+api=b \
-H:+ReportExceptionStackTraces@user \
-H:Features@user+api=org.graalvm.junit.platform.JUnitPlatformFeature \
'-H:Class@explicit main-class=org.graalvm.junit.platform.NativeImageJUnitLauncher' \
-H:ImageBuildID@driver=49c5a172-9667-b6bd-3fe2-c231f3d6deb8 \
'-H:Features@jar:file:///opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/library-support.jar!/META-INF/native-image/com.oracle.svm/thirdparty/native-image.properties+api=com.oracle.svm.thirdparty.gson.GsonFeature' \
'-H:Features@jar:file:///opt/hostedtoolcache/graalvm-jdk-25_linux-x64_bin/25.0.0/x64/graalvm-jdk-25.0.3+9.1/lib/svm/library-support.jar!/META-INF/native-image/com.oracle.svm/polyglot/native-image.properties+api=com.oracle.svm.polyglot.groovy.GroovyIndyInterfaceFeature,com.oracle.svm.polyglot.scala.ScalaFeature'
]
========================================================================================================================
GraalVM Native Image: Generating 'io.quarkus.quarkus-junit-common_tests-tests' (executable)...
========================================================================================================================
For detailed information and explanations on the build output, visit:
https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/BuildOutput.md
------------------------------------------------------------------------------------------------------------------------
[1/8] Initializing...                                                                                    (5.8s @ 0.16GB)
 Java version: 25.0.3+9-LTS, vendor version: Oracle GraalVM 25.0.3+9.1
 Graal compiler: optimization level: b, target machine: x86-64-v3, PGO: off
 C compiler: gcc (linux, x86_64, 13.3.0)
 Garbage collector: Serial GC (max heap size: 80% of RAM)
 2 user-specific feature(s):
 - com.oracle.svm.thirdparty.gson.GsonFeature
 - org.graalvm.junit.platform.JUnitPlatformFeature
------------------------------------------------------------------------------------------------------------------------
 1 experimental option(s) unlocked:
 - '-H:BuildOutputJSONFile' (origin(s): command line)
------------------------------------------------------------------------------------------------------------------------
Build resources:
 - 12.67GB of memory (75.6% of system memory, $CI set to 'true')
 - 4 thread(s) (100.0% of 4 available processor(s), determined at start)
[junit-platform-native] Running in 'test listener' mode using files matching pattern [junit-platform-unique-ids*] found in folder [/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/test-results/test/testlist] and its subfolders.
[2/8] Performing analysis...  [*****]                                                                   (34.8s @ 0.86GB)
    8,853 types,   9,831 fields, and  37,821 methods found reachable
    2,770 types,     280 fields, and   1,486 methods registered for reflection
       63 types,      71 fields, and      55 methods registered for JNI access
        0 downcalls and 0 upcalls registered for foreign access
        4 native libraries: dl, pthread, rt, z
[3/8] Building universe...                                                                               (4.2s @ 0.99GB)
[4/8] Parsing methods...      [**]                                                                       (4.4s @ 0.62GB)
[5/8] Inlining methods...     [***]                                                                      (2.2s @ 0.66GB)
[6/8] Compiling methods...    [****]                                                                    (18.7s @ 0.81GB)
[7/8] Laying out methods...   [**]                                                                       (3.4s @ 0.77GB)
[8/8] Creating image...       [**]                                                                       (3.3s @ 0.88GB)
   9.67MB (35.77%) for code area:    23,455 compilation units
  13.43MB (49.70%) for image heap:  214,956 objects and 151 resources
   3.93MB (14.52%) for other data
  27.03MB in total image size, 24.91MB in total file size
------------------------------------------------------------------------------------------------------------------------
Top 10 origins of code area:                                Top 10 object types in image heap:
   4.75MB java.base                                            2.82MB byte[] for code metadata
   2.12MB svm.jar (Native Image)                               2.65MB byte[] for java.lang.String
 552.96kB assertj-core-3.22.0.jar                              1.38MB java.lang.String
 482.24kB java.xml                                             1.10MB java.lang.Class
 291.01kB junit-jupiter-engine-6.1.0.jar                     920.71kB com.oracle.svm.core.hub.DynamicHubCompanion
 171.40kB junit-platform-launcher-6.1.0.jar                  708.11kB byte[] for general heap data
 164.76kB junit-platform-engine-6.1.0.jar                    449.63kB java.util.concurrent.ConcurrentHashMap$Node
 104.46kB java.logging                                       301.47kB java.util.HashMap$Node
 100.24kB junit-platform-reporting-6.1.0.jar                 285.87kB heap alignment
  97.66kB junit-platform-commons-6.1.0.jar                   282.27kB java.lang.Object[]
 470.57kB for 27 more packages                                 2.54MB for 2237 more object types
                            Use '--emit build-report' to create a report with more details.
------------------------------------------------------------------------------------------------------------------------
Security report:
 - Binary includes Java deserialization.
 - CycloneDX SBOM with 20 component(s) is embedded in binary (1.04kB). 4 type(s) could not be associated to a component.
 - Advanced obfuscation not enabled; enable with '-H:AdvancedObfuscation=""' (experimental support).
------------------------------------------------------------------------------------------------------------------------
Recommendations:
 G1GC: Use the G1 GC ('--gc=G1') for improved latency and throughput.
 PGO:  Use Profile-Guided Optimizations ('--pgo') for improved throughput.
 FUTR: Use '--future-defaults=all' to prepare for future releases.
 HEAP: Set max heap for improved and more predictable memory usage.
 CPU:  Enable more CPU features with '-march=native' for improved performance.
------------------------------------------------------------------------------------------------------------------------
                        5.3s (6.6% of total time) in 728 GCs | Peak RSS: 1.74GB | CPU load: 3.63
------------------------------------------------------------------------------------------------------------------------
Build artifacts:
 /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/native/nativeTestCompile/io.quarkus.quarkus-junit-common_tests-tests (executable)
 /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/native/nativeTestCompile/libmanagement_ext.so (jdk_library)
 /tmp/native-image-build-output.json (build_info)
========================================================================================================================
Finished generating 'io.quarkus.quarkus-junit-common_tests-tests' in 1m 18s.
[native-image-plugin] Native Image written to: /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/native/nativeTestCompile

> Task :nativeTest FAILED
[junit-platform-native] WARNING: -djunit.platform.listeners.uid.tracking.output.dir not specified, trying to find test-ids on default Gradle/Maven locations. As this is a fallback mode, it could take a while. This should only happen if you are running tests executable manually and you didn't pass uid output directory with -djunit.platform.listeners.uid.tracking.output.dir=<path>.
[junit-platform-native] WARNING: Using test-ids from default Gradle project location:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/test-results/test/testlist
Exception in thread "main" com.oracle.svm.core.jdk.UnsupportedFeatureError: SerializationConstructorAccessor class not found for declaringClass: org.junit.platform.engine.UniqueId$SerializedForm (targetConstructorClass: java.lang.Object). Usually adding org.junit.platform.engine.UniqueId$SerializedForm to serialization-config.json fixes the problem.
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.util.VMError.unsupportedFeature(VMError.java:121)
	at org.graalvm.nativeimage.builder/com.oracle.svm.core.reflect.serialize.SerializationSupport.getSerializationConstructorAccessor(SerializationSupport.java:286)

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
17 actionable tasks: 7 executed, 10 up-to-date
	at java.base@25.0.3/jdk.internal.reflect.ReflectionFactory.generateConstructor(ReflectionFactory.java:2439)
	at java.base@25.0.3/jdk.internal.reflect.ReflectionFactory.newConstructorForSerialization(ReflectionFactory.java:297)
	at java.base@25.0.3/java.io.ObjectStreamClass.getSerializableConstructor(ObjectStreamClass.java:1319)
	at java.base@25.0.3/java.io.ObjectStreamClass.<init>(ObjectStreamClass.java:371)
	at java.base@25.0.3/java.io.ObjectStreamClass$Caches$1.computeValue(ObjectStreamClass.java:93)
	at java.base@25.0.3/java.io.ObjectStreamClass$Caches$1.computeValue(ObjectStreamClass.java:90)
	at java.base@25.0.3/java.io.ClassCache$1.computeValue(ClassCache.java:78)
	at java.base@25.0.3/java.io.ClassCache$1.computeValue(ClassCache.java:75)
	at java.base@25.0.3/java.lang.ClassValue.get(JavaLangSubstitutions.java:542)
	at java.base@25.0.3/java.io.ClassCache.get(ClassCache.java:89)
	at java.base@25.0.3/java.io.ObjectStreamClass.lookup(ObjectStreamClass.java:76)
	at java.base@25.0.3/java.io.ObjectStreamClass.lookup(ObjectStreamClass.java:227)
	at org.junit.platform.engine.UniqueId.<clinit>(UniqueId.java:50)
	at org.junit.platform.engine.discovery.DiscoverySelectors.selectUniqueId(DiscoverySelectors.java:1199)
	at java.base@25.0.3/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:214)
	at java.base@25.0.3/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1716)
	at java.base@25.0.3/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:803)
	at java.base@25.0.3/java.util.stream.ReferencePipeline$7$1FlatMap.accept(ReferencePipeline.java:293)
	at java.base@25.0.3/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:214)
	at java.base@25.0.3/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:197)
	at java.base@25.0.3/java.util.Iterator.forEachRemaining(Iterator.java:133)
	at java.base@25.0.3/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1939)
	at java.base@25.0.3/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:570)
	at java.base@25.0.3/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:560)
	at java.base@25.0.3/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:921)
	at java.base@25.0.3/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:265)
	at java.base@25.0.3/java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:723)
	at org.graalvm.junit.platform.NativeImageJUnitLauncher.getSelectors(NativeImageJUnitLauncher.java:166)
	at org.graalvm.junit.platform.NativeImageJUnitLauncher.getTestPlan(NativeImageJUnitLauncher.java:145)
	at org.graalvm.junit.platform.NativeImageJUnitLauncher.main(NativeImageJUnitLauncher.java:119)
	at java.base@25.0.3/java.lang.invoke.LambdaForm$DMH/sa346b79c.invokeStaticInit(LambdaForm$DMH)

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':nativeTest'.
> Process 'command '/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/io.quarkus/quarkus-junit-common/3.33.1/build/native/nativeTestCompile/io.quarkus.quarkus-junit-common_tests-tests'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 1m 22s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for io.quarkus:quarkus-junit-common:3.33.1 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 1m 23s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 3.37.0.CR1 at 2026-06-10T19:16:55Z with exit code 1 after 84s
❌ ERROR: 'native-image run' for 3.37.0.CR1 failed with exit code 1.
FAILED[native-image run][3.37.0.CR1][env GVM_TCK_LV=3.37.0.CR1 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean test -Pcoordinates=io.quarkus:quarkus-junit-common:3.33.1]
```

--- a/metadata/io.quarkus/quarkus-junit-common/3.37.0.CR1/reachability-metadata.json
+++ b/metadata/io.quarkus/quarkus-junit-common/3.37.0.CR1/reachability-metadata.json
@@ -1,0 +1,66 @@
+{
+  "reflection": [
+    {
+      "condition": {
+        "typeReached": "io.quarkus.test.junit.common.ClearCache"
+      },
+      "type": "ch.qos.logback.classic.Logger"
+    },
+    {
+      "condition": {
+        "typeReached": "io.quarkus.test.junit.common.ClearCache"
+      },
+      "type": "com.sun.naming.internal.ResourceManager"
+    },
+    {
+      "condition": {
+        "typeReached": "io_quarkus.quarkus_junit_common.ClearCacheTest"
+      },
+      "type": "io_quarkus.quarkus_junit_common.ClearCacheTest$Markers",
+      "methods": [
+        {
+          "name": "value",
+          "parameterTypes": []
+        }
+      ]
+    },
+    {
+      "condition": {
+        "typeReached": "io.quarkus.test.junit.common.ClearCache"
+      },
+      "type": "org.apache.log4j.LogManager"
+    },
+    {
+      "condition": {
+        "typeReached": "io.quarkus.test.junit.common.ClearCache"
+      },
+      "type": "org.apache.logging.log4j.Logger"
+    },
+    {
+      "condition": {
+        "typeReached": "io.quarkus.test.junit.common.ClearCache"
+      },
+      "type": "org.jboss.logmanager.LogManager"
+    }
+  ],
+  "resources": [
+    {
+      "condition": {
+        "typeReached": "io_quarkus.quarkus_junit_common.ClearCacheTest"
+      },
+      "glob": "META-INF/services/org.assertj.core.configuration.Configuration"
+    },
+    {
+      "condition": {
+        "typeReached": "io_quarkus.quarkus_junit_common.ClearCacheTest"
+      },
+      "glob": "META-INF/services/org.assertj.core.presentation.Representation"
+    },
+    {
+      "condition": {
+        "typeReached": "io.quarkus.test.junit.common.ClearCache"
+      },
+      "glob": "META-INF/services/org.jboss.logging.LoggerProvider"
+    }
+  ]
+}

--- a/metadata/io.quarkus/quarkus-junit-common/3.37.0.CR1/reachability-metadata.json
+++ b/metadata/io.quarkus/quarkus-junit-common/3.37.0.CR1/reachability-metadata.json
@@ -1,66 +1,51 @@
 {
-  "reflection": [
+  "reflection" : [
     {
-      "condition": {
-        "typeReached": "io.quarkus.test.junit.common.ClearCache"
+      "condition" : {
+        "typeReached" : "org.junit.platform.commons.util.KotlinReflectionUtils"
       },
-      "type": "ch.qos.logback.classic.Logger"
+      "type" : "kotlin.Metadata"
     },
     {
-      "condition": {
-        "typeReached": "io.quarkus.test.junit.common.ClearCache"
+      "condition" : {
+        "typeReached" : "io.quarkus.test.junit.common.ClearCache"
       },
-      "type": "com.sun.naming.internal.ResourceManager"
-    },
-    {
-      "condition": {
-        "typeReached": "io_quarkus.quarkus_junit_common.ClearCacheTest"
-      },
-      "type": "io_quarkus.quarkus_junit_common.ClearCacheTest$Markers",
-      "methods": [
+      "type" : "org.junit.platform.commons.util.AnnotationUtils",
+      "fields" : [
         {
-          "name": "value",
-          "parameterTypes": []
+          "name" : "repeatableAnnotationContainerCache"
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "io.quarkus.test.junit.common.ClearCache"
+      "condition" : {
+        "typeReached" : "io.quarkus.test.junit.common.ClearCache"
       },
-      "type": "org.apache.log4j.LogManager"
-    },
-    {
-      "condition": {
-        "typeReached": "io.quarkus.test.junit.common.ClearCache"
-      },
-      "type": "org.apache.logging.log4j.Logger"
-    },
-    {
-      "condition": {
-        "typeReached": "io.quarkus.test.junit.common.ClearCache"
-      },
-      "type": "org.jboss.logmanager.LogManager"
+      "type" : "com.sun.naming.internal.ResourceManager",
+      "fields" : [
+        {
+          "name" : "propertiesCache"
+        }
+      ]
     }
   ],
-  "resources": [
+  "resources" : [
     {
-      "condition": {
-        "typeReached": "io_quarkus.quarkus_junit_common.ClearCacheTest"
-      },
-      "glob": "META-INF/services/org.assertj.core.configuration.Configuration"
+      "glob" : "META-INF/services/org.junit.platform.commons.support.scanning.ClasspathScanner"
     },
     {
-      "condition": {
-        "typeReached": "io_quarkus.quarkus_junit_common.ClearCacheTest"
+      "condition" : {
+        "typeReached" : "io.quarkus.test.junit.common.ClearCache"
       },
-      "glob": "META-INF/services/org.assertj.core.presentation.Representation"
-    },
+      "glob" : "META-INF/services/org.jboss.logging.LoggerProvider"
+    }
+  ],
+  "serialization" : [
     {
-      "condition": {
-        "typeReached": "io.quarkus.test.junit.common.ClearCache"
+      "condition" : {
+        "typeReached" : "org.junit.platform.engine.UniqueId"
       },
-      "glob": "META-INF/services/org.jboss.logging.LoggerProvider"
+      "type" : "org.junit.platform.engine.UniqueId$SerializedForm"
     }
   ]
 }

--- a/metadata/io.quarkus/quarkus-junit-common/index.json
+++ b/metadata/io.quarkus/quarkus-junit-common/index.json
@@ -13,7 +13,8 @@
     ],
     "allowed-packages" : [
       "io.quarkus.test.junit.common",
-      "org.junit.platform.commons.util"
+      "org.junit.platform.commons.util",
+      "org.junit.platform.engine"
     ]
   },
   {

--- a/metadata/io.quarkus/quarkus-junit-common/index.json
+++ b/metadata/io.quarkus/quarkus-junit-common/index.json
@@ -1,6 +1,22 @@
 [
   {
     "latest" : true,
+    "metadata-version" : "3.37.0.CR1",
+    "test-version" : "3.33.1",
+    "source-code-url" : "https://repo.maven.apache.org/maven2/io/quarkus/quarkus-junit-common/$version$/quarkus-junit-common-$version$-sources.jar",
+    "repository-url" : "https://github.com/quarkusio/quarkus",
+    "test-code-url" : "N/A",
+    "documentation-url" : "https://repo.maven.apache.org/maven2/io/quarkus/quarkus-junit-common/$version$/quarkus-junit-common-$version$-javadoc.jar",
+    "description" : "Quarkus JUnit Common provides shared testing support classes used by Quarkus JUnit test framework modules. It contains common JUnit integration utilities and execution conditions that are reused by the internal and public Quarkus JUnit artifacts.",
+    "tested-versions" : [
+      "3.37.0.CR1"
+    ],
+    "allowed-packages" : [
+      "io.quarkus.test.junit.common",
+      "org.junit.platform.commons.util"
+    ]
+  },
+  {
     "metadata-version" : "3.33.1",
     "source-code-url" : "https://repo.maven.apache.org/maven2/io/quarkus/quarkus-junit-common/$version$/quarkus-junit-common-$version$-sources.jar",
     "repository-url" : "https://github.com/quarkusio/quarkus",

--- a/stats/io.quarkus/quarkus-junit-common/3.37.0.CR1/execution-metrics.json
+++ b/stats/io.quarkus/quarkus-junit-common/3.37.0.CR1/execution-metrics.json
@@ -1,0 +1,127 @@
+{
+  "fix_ni_run:2026-06-10": {
+    "timestamp": "2026-06-10T23:29:11.810818Z",
+    "library": "io.quarkus:quarkus-junit-common:3.37.0.CR1",
+    "starting_commit": "b1d3ecc76b4e34a557e7b81d977303aa2f5b2e95",
+    "ending_commit": "98b5b84c9802664c023ee12faed49ba5fa51752a",
+    "previous_library": "io.quarkus:quarkus-junit-common:3.33.1",
+    "strategy_name": "library_update_pi_gpt-5.5",
+    "agent": "pi",
+    "model": "oca/gpt-5.5",
+    "status": "success",
+    "stats": {
+      "version": "3.37.0.CR1",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 2,
+            "totalCalls": 2
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 2,
+        "totalCalls": 2
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 43,
+          "missed": 111,
+          "ratio": 0.279221,
+          "total": 154
+        },
+        "line": {
+          "covered": 20,
+          "missed": 34,
+          "ratio": 0.37037,
+          "total": 54
+        },
+        "method": {
+          "covered": 6,
+          "missed": 7,
+          "ratio": 0.461538,
+          "total": 13
+        }
+      }
+    },
+    "previous_library_stats": {
+      "version": "3.33.1",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 2,
+            "totalCalls": 2
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 2,
+        "totalCalls": 2
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 43,
+          "missed": 87,
+          "ratio": 0.330769,
+          "total": 130
+        },
+        "line": {
+          "covered": 20,
+          "missed": 26,
+          "ratio": 0.434783,
+          "total": 46
+        },
+        "method": {
+          "covered": 6,
+          "missed": 5,
+          "ratio": 0.545455,
+          "total": 11
+        }
+      }
+    },
+    "library_preparation_preflight": {
+      "status": "degraded",
+      "action": "no_action",
+      "issue_number": 8330,
+      "issue_label": "fails-native-image-run",
+      "library": "io.quarkus:quarkus-junit-common:3.37.0.CR1",
+      "summary": "Library preparation preflight did not produce usable advisory setup.",
+      "deterministic_setup": [],
+      "agent_guidance": "",
+      "risks": [],
+      "applied_setup": [],
+      "input_evidence": [
+        {
+          "name": "issue",
+          "summary": "#8330 [Automation] native-image run fails for io.quarkus:quarkus-junit-common:3.37.0.CR1; label=fails-native-image-run; body_chars=8015"
+        },
+        {
+          "name": "existing_tests",
+          "summary": "6 existing test file path(s) included"
+        }
+      ],
+      "current_library": "io.quarkus:quarkus-junit-common:3.33.1",
+      "new_version": "3.37.0.CR1",
+      "failure_reason": "FileNotFoundError: [Errno 2] No such file or directory",
+      "model": "oca/gpt-5.5",
+      "prompt_path": "library-preflight-prompt.txt",
+      "session_log_path": "/home/jovan/Work/graalvm-reachability-metadata/forge/logs/io.quarkus:quarkus-junit-common:3.37.0.CR1/library-preparation-preflight/pi-generation-2026-06-10T23-12-11-723088Z.log"
+    },
+    "metrics": {
+      "input_tokens_used": 0,
+      "cached_input_tokens_used": 0,
+      "output_tokens_used": 0,
+      "iterations": 0,
+      "cost_usd": 0.0,
+      "tested_library_loc": 20,
+      "metadata_entries": 7,
+      "previous_library_metadata_entries": 7,
+      "code_coverage_percent": 37.04,
+      "previous_library_coverage_percent": 43.48
+    },
+    "artifacts": {
+      "test_file": "tests/src/io.quarkus/quarkus-junit-common/3.33.1/src/test/java/io_quarkus/quarkus_junit_common/ClearCacheTest.java",
+      "metadata_file": "metadata/io.quarkus/quarkus-junit-common/3.37.0.CR1/reachability-metadata.json"
+    }
+  }
+}

--- a/stats/io.quarkus/quarkus-junit-common/3.37.0.CR1/stats.json
+++ b/stats/io.quarkus/quarkus-junit-common/3.37.0.CR1/stats.json
@@ -1,0 +1,37 @@
+{
+  "versions" : [ {
+    "version" : "3.37.0.CR1",
+    "dynamicAccess" : {
+      "breakdown" : {
+        "reflection" : {
+          "coverageRatio" : 1.0,
+          "coveredCalls" : 2,
+          "totalCalls" : 2
+        }
+      },
+      "coverageRatio" : 1.0,
+      "coveredCalls" : 2,
+      "totalCalls" : 2
+    },
+    "libraryCoverage" : {
+      "instruction" : {
+        "covered" : 43,
+        "missed" : 111,
+        "ratio" : 0.279221,
+        "total" : 154
+      },
+      "line" : {
+        "covered" : 20,
+        "missed" : 34,
+        "ratio" : 0.37037,
+        "total" : 54
+      },
+      "method" : {
+        "covered" : 6,
+        "missed" : 7,
+        "ratio" : 0.461538,
+        "total" : 13
+      }
+    }
+  } ]
+}

